examples/Greeter/Client/Client.csproj

```diff
@@ -6,12 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Protobuf Include="..\..\Proto\greet.proto" GrpcServices="Client" Link="Protos\greet.proto" />
+    <Protobuf Include="..\Proto\greet.proto" GrpcServices="Client" Link="Protos\greet.proto" />
 
     <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPackageVersion)" />
+    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcDotNetPackageVersion)" />
```

Comment:
Curious why use the package instead of the project?

Reply:
Because these samples are designed to be as independent as possible (yes the variable could also be hardcoded but I didn't want to duplicate it everywhere)